### PR TITLE
Static analysis revealed

### PR DESCRIPTION
**Pull request by szepeviktor**
Powered by @phpstan
See https://packagist.org/packages/szepeviktor/phpstan-wordpress

### CODE REVIEW

**Comment by szepeviktor** [comment]
@aristath Could you merge this one?

File: src/Notices.php [hunk 1]
```diff
@@ -87,6 +87,7 @@ public function get( $id ) {
 		if ( isset( $this->notices[ $id ] ) ) {
 			return $this->notices[ $id ];
 		}
+		return null;
```
Comment (aristath) [inline src/Notices.php:90]: @szepeviktor Why was this line added?
Functions without a return statement will return `null` at runtime anyway :thinking:

Comment (szepeviktor) [reply src/Notices.php:90]: @aristath Implicit programming **vs.** Explicit programming -> explicit wins! 
🥇  Humans can read it and it's testable.

**Comment by pattonwebz** [comment]
@aristath this looks good to me, any objections to me merging this? Are you ok with explicit `null` return in that method?

**Comment by aristath** [comment]
No objections, this one just slipped through the cracks. Merging :+1:

**Comment by szepeviktor** [comment]
Thank you.
